ethereum/referencetests/src/main/java/org/hyperledger/besu/ethereum/referencetests/ReferenceTestBlockchain.java

```diff
@@ -166,4 +167,15 @@ public boolean removeChainReorgObserver(final long observerId) {
       super(message);
     }
   }
+
+  @Override
+  public Comparator<BlockHeader> getBlockChoiceRule() {
+    return (a, b) -> {
+      throw new NonDeterministicOperationException(
+          "ReferenceTestBlockchian for VMTest Chains do not support fork choice rules");
+    };
+  }
+
+  @Override
+  public void setBlockChoiceRule(final Comparator<BlockHeader> blockChoiceRule) {}
```

Comment:
Could we add an exception here too? Or does it need to be a noop for things to work?

Reply:
Sure.  Assuming CI doesn't break.